### DISCUSSION TITLE
Are IP Adresses allowed in Labels?

### DISCUSSION THREAD

**Discussion by florknts** [grafana/mimir#6236] [Help and support]
Hi there,

I'm pretty new to Grafana and Mimir, and i read in Prometheus data model, that only the following symbols are allowed in labels: 
[a-zA-Z_:][a-zA-Z0-9_:]*

https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels

Since I've heard Prometheus and Mimir are very alike considering Labels. 

Am I allowed to use IP Adresses for e.g. 172.12.1.1 in my label? 

Help is much appreciated and thank you in advance,
Florian

**Comment by dimitarvdimitrov** [comment]
`172.12.1.1` is not a valid label __name__, but it's a perfectly valid label __value__. That regular expression is only for validating the label names.
